Dfe.Academies.External.Web/AcademiesAPIResponseModels/Schools/EstablishmentResponse.cs

```diff
@@ -1,5 +1,3 @@
-using System.Text.Json.Serialization;
-
 namespace Dfe.Academies.External.Web.AcademiesAPIResponseModels.Schools;
 
 public record EstablishmentResponse
```

Comment:
There are nullable related warnings in this class, I think we could resolve this by removing the default constructor and changing the implementation of the ResilientRequestProvider to use:

```
return await _client.GetFromJsonAsync<TResult>(uri);
```

Reply:
?? can we catch up about this tomo @wystanpalm-madetech ?

Reply:
Amended ResilientRequestProvider Get<> method @wystanpalm-madetech